### PR TITLE
[ZL-18] add official account with access token

### PR DESCRIPTION
**Pull request by nguyenvansy**
### Release Note
- Add official account with access token

### PR Check (for Reviewer)
- [x] "How to Confirm" is Correct

### How to confirm
1. Login with business user
2. Go to http://localhost:3000/official
3. Click `Create new` button that is next to `Search` field
4. Please input `access token` and click `Show info` button
```
oDgkV8rrsJtDqQzuX7BRDSVNh72P1ev0gjh_HhWkYbE_zVj5kL7PNe3rx7wVBU5Wf_R19BLMi1ZozeSerWwFATBDf1R04fDLrCZZRPDirM2IgCnoam_eFg6jcXAGNUebheJ3AET1onBzckC8l3_BJAdcz5gKE8HFfvsZ0wrdq0QkmgGrlmIE4w2hfGEA3O8Jj_sABAeOZWcKtwutl7s9CBg3YHJoOObLmAUvR8H4ctJPeEr4c4_xGzdjx53AEUTrxF76OEKkocVXYhfnmsIHP72NHdIBOzqx
```
5. Please confirm that OA info showed below
6. Click `Save` button and confirm that Official Account saved well in DB :+1: 

image

### CODE REVIEW

File: client/src/Components/pages/Official.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
 import React , { useState, useEffect } from 'react'
```
Comment (Tu-LeVan) [inline client/src/Components/pages/Official.js]: Please remove a space

File: client/src/Components/pages/official_account/OfficialAccountCreate.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+import { Input, Button, Grid } from 'semantic-ui-react'
+import OfficialAccountDetail from './OfficialAccountDetail'
+import ErrorMessage from '../../ErrorMessage'
+import ZolaModal from '../../ZolaModal'
+import { createOfficialAccount, exchangeZaloOAInfo } from '../../../resolvers/officialAccount'
+
+const OfficialAccountCreate = () => {
+  const [token, setToken] = useState('')
+  const [infoVisible, setInfoVisible] = useState(false)
```
Comment (Tu-LeVan) [inline client/src/Components/pages/official_account/OfficialAccountCreate.js]: Don't need to use this state.

File: client/src/Components/ZolaModal.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Modal } from 'semantic-ui-react'
+import { removeErrors } from '../reducers/errorsReducer'
+
+const ZolaModal = ({title, visible, onClose, content, actions, errors, removeErrors }) => {
```
Comment (Tu-LeVan) [inline client/src/Components/ZolaModal.js]: Add a space behind `title`.

File: client/src/styles/search.css [hunk 4]
```diff
@@ -1,6 +1,5 @@
 .search-bar {
-  display: flex;
-  justify-content: flex-end;
+  margin-left: 5px;
 }
```
Comment (Tu-LeVan) [inline client/src/styles/search.css]: Add a new line

**Comment by Tu-LeVan** [comment]
@nguyenvansy san LGTM :+1: 
@hientuminh san Please check it as the second review.

File: client/src/resolvers/officialAccount.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  const errors = store.getState().errors
+  if (errors && errors.length > 0) store.dispatch(removeErrors())
+}
+
+export const ALL_OFFICIAL_ACCOUNT_OF_USER = gql`
+{
+  allOfficialAccountOfUser {
+    id
+    accountName
+    accountToken
```
Comment (hientuminh) [inline client/src/resolvers/officialAccount.js]: Please remove `accountToken`

File: client/src/resolvers/officialAccount.js [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  ) {
+    createOfficialAccount(
+      accountName: $accountName
+      accountToken: $accountToken
+      description: $description
+      avatar: $avatar
+    ) {
+      id
+      accountName
+      accountToken
```
Comment (hientuminh) [inline client/src/resolvers/officialAccount.js]: Please remove `accountToken`

**Comment by hientuminh** [comment]
@nguyenvansy @Tu-LeVan LGTM 👍

**Pull request merged**